active/ListViewGridView/listviewgridview-designupdates.md

```diff
@@ -0,0 +1,397 @@
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxDisabledBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxSelectedBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxSelectedPointerOverBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxSelectedPressedBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxSelectedDisabledBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxBorderBrushProperty{ get; };`
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxPointerOverBorderBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxPressedBorderBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxDisabledBorderBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckPressedBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckDisabledBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.SelectedInnerBorderBrushProperty{ get; };`
+
+
+## New Visual State Group and States:
+
+### For reference, existing visual group and states for the SelectionMode==Multiple option: 
+
+**VisualStateGroup MultiSelectStates**
+
+VisualState MultiSelectDisabled  for selectionMode != Multiple
+
+VisualState MultiSelectEnabled   for selectionMode == Multiple
+
+
+### New visual group and states for the Selection==Single or Extended options: 
+
+**VisualStateGroup SelectionIndicatorStates**
+
+VisualState SelectionIndicatorDisabled  for selectionMode == None or Multiple
+
+VisualState SelectionIndicatorEnabled   for selectionMode == Single or Extended
+
+## Changes to existing APIs:
+All of the following APIs will affect the rounded backplate of the item, rather than the overall footprint as they did prior. 
+
+`public Windows.UI.Xaml.Media.Brush ListViewItemPresenter.PointerOverBackground { get; set; }`
+
+`public Windows.UI.Xaml.Media.Brush ListViewItemPresenter.SelectedBackground { get; set; }`
+
+`public Windows.UI.Xaml.Media.Brush ListViewItemPresenter.SelectedPointerOverBackground { get; set; }`
+
+`public Windows.UI.Xaml.Media.Brush ListViewItemPresenter.PressedBackground { get; set; }`
+
+`public Windows.UI.Xaml.Media.Brush ListViewItemPresenter.SelectedPressedBackground { get; set; }`
+
+## New ThemeResources
+
+### ListView:
+
+```xml
+<ListViewItemPresenterSelectionIndicatorMode x:Key="ListViewItemSelectionIndicatorMode">Inline</ListViewItemPresenterSelectionIndicatorMode>
+
+<x:Boolean x:Key="ListViewItemSelectionIndicatorVisualEnabled">True</x:Boolean>
+
+<StaticResource x:Key="ListViewItemSelectionIndicatorDisabledBrush" ResourceKey="SystemAccentColor"/>
```

Comment:
Assuming these **SystemAccentColor** references are filler until you finalise the assignment of the new CommonResource brushes?

Reply:
Yes.